tasks/kmt.py

```diff
@@ -381,6 +382,36 @@ def test(ctx, stack=None, packages="", run=None, retry=2, rebuild_deps=False, vm
     )
 
 
+@task
+def build(ctx, stack=None, vms="", ssh_key="", rebuild_deps=False):
```

Comment:
Is `vms` required? If so, I think putting it at the end and not providing a default will make it so.